build.sbt

```diff
@@ -497,8 +497,7 @@ lazy val finagleException = Project(
     name := "finagle-exception",
     libraryDependencies ++= Seq(
       util("codec")
-    ) ++ scroogeLibs,
-    libraryDependencies ++= jacksonLibs
+    ) ++ scroogeLibs
```

Comment:
finagle-exception uses jackson directly, so it really should have a dependency on jackson.